app/models/game.rb

```diff
@@ -252,4 +297,7 @@ def self.missing_games
     )
   end
 
-end
+  def self.stored_games
+    where(status: Game.statuses[:storage])
+  end
+end
```

Comment:
Fix your whitespace errors! :)

Reply:
Sorry, what's the whitespace error here? Thanks!

Reply:
New line at end of file is missing.